isbn/lib.php

```diff
@@ -69,7 +69,8 @@
     'produktSigel' => '//feld[@nr="078" and @ind="e"]/uf[@code="a"]',
 );
 
-function performMapping($map, $outputXml) {
+function performMapping($map, $outputXml)
+{
```

Comment:
Okay. I guess I am not so much used to that... I think JavaScript is doing that differently...